### PR TITLE
feat: Enable personalization call for Upfront Presentment

### PR DESCRIPTION
**Pull request by akshay-kr**
This PR enables personalization call for Upfront Presentment tagline.

### CODE REVIEW

File: server/service/personalization.jsx [hunk 1]
```diff
@@ -127,8 +130,14 @@ function contentToJSX(content : string) : ComponentFunctionType<PersonalizationC
 }
 
 export async function resolvePersonalization(req : ExpressRequest, gqlBatch : GraphQLBatchCall, personalizationOptions : PersonalizationOptions) : Promise<Personalization> {
+    // $FlowFixMe
+    const personalizationEnabled = req.app && req.app.kraken && req.app.kraken.get('ui:enablePersonalization');
```
Comment (gregjopa) [inline server/service/personalization.jsx:134]: @akshay-kr Let's move this logic over to the parent app and pass this down as an argument to the mounted app.

Comment (akshay-kr) [reply server/service/personalization.jsx]: Done. Moved the logic to parent app.

**Comment by akshay-kr** [comment]
@gregjopa Addressed review comment. Please review again.

File: server/service/personalization.jsx [hunk 2]
```diff
@@ -137,12 +145,13 @@ export async function resolvePersonalization(req : ExpressRequest, gqlBatch : Gr
     intent = intent ? intent.toUpperCase() : intent;
     label = label ? label.toUpperCase() : label;
 
+    const taglineEnabled = tagline === true || tagline === 'true';
```
Comment (bluepnume) [inline server/service/personalization.jsx:148]: Can we get a 100ms timeout for this call? Want to make sure it doesn't block button render if GQL latency spikes for any reason.

Comment (akshay-kr) [reply server/service/personalization.jsx:149]: Done. Added timeout of 100ms.

**Review by gregjopa — APPROVED** [review]
Looks good!

**Pull request merged**